models/chainlink/ccip/chainlink_ccip_schema.yml

```diff
@@ -0,0 +1,73 @@
+version: 2
+
+models:
```

Comment:
`chainlink_ccip_events` may be missing in this schema.yml

Reply:
Will add along with the items in the comment ^. 

Thanks for the help here!